lldb/tools/lldb-vscode/VSCode.cpp

```diff
@@ -418,7 +419,9 @@ ExpressionContext VSCode::DetectExpressionContext(lldb::SBFrame &frame,
         if (!auto_repl_mode_collision_warning) {
           llvm::errs() << "Variable expression '" << text
                        << "' is hiding an lldb command, prefix an expression "
-                          "with ` to ensure it runs as a lldb command.\n";
+                          "with "
+                       << g_vsc.command_escape_prefix
+                       << " to ensure it runs as a lldb command.\n";
```

Comment:
Since the expression prefix can be empty we should be quotes around this if it isn't empty, and maybe not show this at all if g_vsc.command_escape_prefix is empty?